### ISSUE TITLE
Please explain plugin usefulness

### ISSUE THREAD

**Issue by pcav** [schulzsebastian/qgisplugin_spreadsheet#1]
now that we can load xls and ods files directly from the browser.
Thanks.

**Comment by schulzsebastian** [comment]
@pcav We can load only attributes table, without geometry. Here is an option to load (and convert, eg from degrees in WGS84) geometry using columns with coordinates and export it to Shapefile or memory layer. Other related additional functionalities will appear soon.

**Comment by pcav** [comment]
Thanks for the clarification; so, isn't it similar to http://plugins.qgis.org/plugins/SpreadsheetLayers/ ?
There may be more similar plugins as well.
Sorry to bother, we're striving hard to reduce duplication, and create larger development groups for plugins. I hope you understand.

**Comment by schulzsebastian** [comment]
@pcav That's all right, I understand. Maybe the topic is the same but idea and plugins are different. SpreadsheetLayers creates a VRT file (which is only a relation based on SQLite with file). There is no option to **edit** or **export** spatial data.

**Comment by pcav** [comment]
@schulzsebastian thanks for the analysis. Would you mind checking the scope for merging the two, possibly searching if there are more similar plugins?
Thanks.

**Comment by NyakudyaA** [comment]
@schulzsebastian What is the status of this?
